### ISSUE TITLE
[plug-in] window.onDidOpenTerminal is not implemented

### ISSUE THREAD

**Issue by vitaliy-guliy** [theia-ide/theia#4222]
The theia.window.onDidOpenTerminal Plugin API is declared but still not implemented.
Plugin fails when it tries to use theia.window.onDidOpenTerminal.

**Comment by akosyakov** [comment]
Are you sure? https://github.com/theia-ide/theia/blob/6b1a35cf249dc8c6c73cdf058cba4c5673b7e31c/packages/plugin-ext/src/plugin/plugin-context.ts#L310

**Comment by vitaliy-guliy** [comment]
I get an error in my plugin

![window ondidopenterminal](https://user-images.githubusercontent.com/1655894/52123137-d4f94300-262d-11e9-8eb2-4e695b79c63e.png)

Handle closing the terminals runs without errors
```typescript
export async function start(context: theia.PluginContext) {
    theia.window.onDidCloseTerminal(e => {
        theia.window.showInformationMessage('Terminal has been closed');
    });
}
```

**Comment by vitaliy-guliy** [comment]
But it looks it's implemented. Then, we need to check. It could be because Theia 0.3.18 which we are using in our development does not have those code.

**Comment by AndrienkoAleksandr** [comment]
@vitaliy-guliy it was implemented, but available from 0.3.19 Theia. You think You used 0.3.18.

**Comment by vitaliy-guliy** [comment]
I was used wrong version of Theia in my plugin.
Now everything's fine.
